### PR TITLE
hw 8 was added

### PR DESCRIPTION
**Pull request by serhii-k-n**

### CODE REVIEW

File: Homework_8/dictionary.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+"""
+Задача 1: Є система, де є списки користувачів.
+roles = { 'admin' : [...], 'maintainer' : [...], 'manager' : [...], 'developer': [...]}
+В цих списках треба понаписувати імена користувачів.
+При введенні імені необхідно перевірити, до якої ролі належить ця людина та вивести повідомлення виду: "Hello, <role>"
+Якщо імені немає у списках, вивести: "Hello, Guest"
+"""
+
+roles = {'admin': ['Sergey', 'Ethan'], 'manager': ['Anna', 'Helen'], 'developer': ['Sasha', 'Den']}
+user_name = input('Print your name here: ')
```
Comment (MarinaZelenska) [inline Homework_8/dictionary.py]: I suppose can be better -> user_name = input('Print your name here: ').title().

File: Homework_8/dictionary.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+char_count = {}
+
+for char in result_link:
+    if char in char_count:
+        char_count[char] += 1
+    else:
+        char_count[char] = 1
+
+for char, count in char_count.items():
+    print(f'Symbol "{char}" meets {count} time(s) in string: {result_link}')
```
Comment (MarinaZelenska) [inline Homework_8/dictionary.py:56]: This loop is extra but ok

File: Homework_8/dictionary.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+{'h': 2, 't': 7, 'p': 1} ........
+"""
+
+char_count = {}
+
+for char in result_link:
+    if char in char_count:
+        char_count[char] += 1
+    else:
+        char_count[char] = 1
```
Comment (MarinaZelenska) [inline Homework_8/dictionary.py]: Can be better
image

**Review by MarinaZelenska — APPROVED** [review]

File: Homework_8/dictionary.py (hunk 1 above)
Comment (serhii-k-n) [reply Homework_8/dictionary.py]: agree, done

File: Homework_8/dictionary.py (hunk 2 above)
Comment (serhii-k-n) [reply Homework_8/dictionary.py:56]: yes, i know, i thought would be ok

File: Homework_8/dictionary.py (hunk 3 above)
Comment (serhii-k-n) [reply Homework_8/dictionary.py]: got it, agree, done

**Pull request merged**